src/main/scala/com/microsoft/hyperspace/index/rules/RuleUtils.scala

```diff
@@ -216,25 +226,46 @@ object RuleUtils {
     var unhandledAppendedFiles: Seq[Path] = Nil
```

Comment:
could we please move this variable closer to its usage? Also please add a description for this variable next to declaration?